docs/jwst/references_general/refpix_reffile.inc

```diff
@@ -47,12 +47,52 @@ beta_2   float32
 beta_3   float32
 =======  =========
 
-The "alpha" arrays contain correction multipliers to the data,
-and the "beta" arrays contain correction multipliers to the reference
-output. Both arrays have 4 components; one for each sector.
+The "alpha" arrays contain correction multipliers to the reference output,
+and the "beta" arrays contain correction multipliers to the interleaved
+reference pixels. Both arrays have 4 components, one for each sector
+(amplifier output).  These are applied to (i.e. multiplied by) the Fourier
+transform of the interleaved reference pixel data.
 The coefficients are intrinsically complex values, but have their
 real and imaginary parts stored in alternating table rows, i.e. row 1
 contains the real components of all coefficients and row 2 contains
 the corresponding imaginary components for each.
 This storage scheme results in a total of 2916352 (2048 * 712 * 2)
 rows in the table.
+
+The second BINTABLE extension is labeled with EXTNAME = "DQ" and has the
+following column characteristics:
+
+=======   =========
+Column    Data type
+=======   =========
+output    int16
+odd_even  int16
+mask      uint32
+=======   =========
+
+This table has eight rows.  The "output" column contains the amplifier output
+numbers:  1, 1, 2, 2, 3, 3, 4, 4.  The "odd_even" column contains values
+1 or 2, indicating that either the first or second pair of reference pixel
+reads respectively should be regarded as bad.  The "mask" column contains
+32-bit unsigned integer values.  The interpretation of these values was
+described in the ESA CDP3 document as follows:
+
+"There is also a DQ extension that holds a binary table with three
+columns (OUTPUT, ODD_EVEN, and MASK) and eight rows. In the current
+IRS2 implementation, one jumps 32 times to odd and 32 times to even
+reference pixels, which are then read twice consecutively. Therefore,
+the masks are 32 bit unsigned integers that encode bad interleaved
+reference pixels/columns from left to right (increasing column index)
+in the native detector frame. When a bit is set, the corresponding
+reference data should not be used for the correction."
+
+I assumed that "native detector frame" in the above description referred to
```

Comment:
We should probably change this to be less personal (i.e. not first-person). So perhaps "We assume that ..."